pkg/service/controller_test.go

```diff
@@ -790,13 +791,7 @@ func (c *ControllerClientMock) ListVirtualMachines(_ context.Context, namespace
 				Name:      testVMName,
 				Namespace: namespace,
 			},
-			Spec: kubevirtv1.VirtualMachineInstanceSpec{
-				Domain: kubevirtv1.DomainSpec{
-					Firmware: &kubevirtv1.Firmware{
-						UUID: types.UID(testVMUID),
-					},
-				},
-			},
+			Spec: kubevirtv1.VirtualMachineInstanceSpec{},
```

Comment:
Do we even care about the Spec at this point? Just don't define it?

Reply:
right